### DISCUSSION TITLE
Does this plugin support extend Line Controller?

### DISCUSSION THREAD

**Discussion by christhofer** [victorgarciaesgi/vue-chart-3#67] [Q&A]
I'm creating a percentile graph and I want to have a label right after the end of the line.

This question has accepted answer that we can do this with extend
https://stackoverflow.com/questions/49386612/how-to-add-text-at-end-of-each-line-in-charts-js

But I can't find any mention about extend in this plugin documentation.

**Comment by victorgarciaesgi** [comment]
The accepted answer is using Chart.js 2.x. `vue-chart-3` is only a adapter of Chart.js 3.x for Vue, so it doesn't add features. You can do it the way the docs of Chart.js tells you too
